### PR TITLE
Using generator for wmts tests

### PR DESCRIPTION
**Pull request by procrastinatio**
If you want to use generator, you may note subclass `unititests`

Use it with:

`./buildout/bin/nosetests chsdi/tests/mapproxy/test_wmtsgettile.py:TestWmtsGetTile`

### CODE REVIEW

File: chsdi/tests/mapproxy/test_wmtsgettile.py [hunk 1]
```diff
@@ -53,14 +59,17 @@ def check_status_code(self, path):
             url = path.replace('http://wmts10.geo.admin.ch', self.mapproxy_url)
         url = rotateUrl(url)
         h = get_header()
-        resp = requests.get(url, headers=h['Header'])
+        self.session.headers.update(h['Header'])
+        resp = self.session.get(url)
         checkcode = resp.status_code in h['Results']
-        self.failUnless(checkcode, 'Called Url: ' + url + ' [returned with ' + str(resp.status_code) + ']')
+
+        assert(checkcode)
```
Comment (gjn) [inline chsdi/tests/mapproxy/test_wmtsgettile.py]: Do we still see URL that failed?

**Comment by gjn** [comment]
On jenkins, sitemap tests fail reguraleyl. It's maybe related. Can you have a look as well and integrate into this PR?

**Comment by loicgasser** [comment]
@gjn 
my solution for sitemaps https://github.com/geoadmin/mf-chsdi3/pull/1567

**Comment by procrastinatio** [comment]
jenkins test

**Comment by loicgasser** [comment]
Thanks

**Pull request merged**